api/lib/config.js

```diff
@@ -73,7 +73,7 @@ module.exports = (function () {
     logging: {
       enabled: isFeatureEnabled(process.env.LOG_ENABLED),
       logLevel: process.env.LOG_LEVEL || 'info',
-      logForHumans: isFeatureEnabled(process.env.LOG_FOR_HUMANS),
+      logForHumans: isFeatureEnabled(process.env.LOG_FOR_HUMANS) || process.stdout.isTTY,
```

Comment:
Je propose d'inliner car les opérateur logiques + isFeatureEnabled c'est :exploding_head: 
```suggestion

			getLogForHumans = ()=>{
				  const processOutputingToTerminal = process.stdout.isTTY;
					if (process.env.LOG_FOR_HUMANS === 'true'){ 
						  return true
					}
					if (process.env.LOG_FOR_HUMANS === 'false' && processOutputingToTerminal){
						  return true
					}
					return false
				}
			
      logForHumans: getLogForHumans();
```